server/requirements.txt

```
@@ -2,3 +2,6 @@ overpy>=0.7
 pandas>=2.2.3
 pydantic>=2.10.6
 requests>=2.32.3
+networkx>=3.4.2
+pyproj >= 3.7.1
+shapely >= 2.0.7
```

Please also remove the whitespaces here, they conflict with the rest of the file
```suggestion
pyproj>=3.7.1
shapely>=2.0.7
```

---

👍